src/main/kotlin/domain/MineSweeperGame.kt

```diff
@@ -0,0 +1,43 @@
+package domain
+
+object MineSweeperGame {
+    fun initBoard(weight: Int, height: Int, mineCount: Int): Map<Position, Block> {
+        val board = mutableMapOf<Position, Block>()
+        val safetyBlockCount = weight * height - mineCount
+        val blocks = getBlocks(mineCount, safetyBlockCount)
+        var count = 0
+
+        (0 until weight).forEach { x ->
+            count = initRow(height, board, x, blocks, count)
+        }
+
+        return board.toMap()
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        val safetyBlockCount = weight * height - mineCount
        val blocks = getBlocks(mineCount, safetyBlockCount)

        return blocks.mapIndexed { index, block ->
            val x = index / weight
            val y = index % weight
            Position.getInstance(x, y) to block
        }.toMap()
```

Reply:
이렇게 깔끔하게 할 수 있었네요.. 감사합니다 😭